bedrock/firefox/templates/firefox/all.html

```diff
@@ -75,6 +93,8 @@
 {% block site_header_logo %}
   {% if platform == 'desktop' and channel == 'alpha' %}
     <h2><a href="{{ url('firefox.developer.index') }}">{{ high_res_img('logos/firefox/developer-wordmark-dark.png', {'alt': _('Firefox Developer Edition'), 'width': '260', 'height': '48'}) }}</a></h2>
+  {% elif platform == 'desktop' and channel == 'beta' %}
+    <h2>{{ high_res_img('/protocol/img/logos/firefox/browser/beta/logo-word-hor-md.png', {'alt': 'Firefox Beta', 'width': '260', 'height': '48'}) }}</h2>
```

Comment:
This is throwing an error [on demo](https://bedrock-demos-pr-7920.herokuapp.com/en-US/firefox/beta/all/): `ValueError: Missing staticfiles manifest entry for '/protocol/img/logos/firefox/browser/beta/logo-word-hor-md.png'`

See https://github.com/mozilla/bedrock/issues/7863